### DISCUSSION TITLE
244

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#808] [Comments]
http://jojoldu.tistory.com/244

**Comment by jojoldu** [comment]
작성자: 잘생긴팝콘

9억건들어있는 테이블  컬럼추가하나에 6시간걸렸는데 담엔 이방법해보겠습니다

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

넵 적극 추천드립니다^^

**Comment by kimsangkyun** [comment]
궁금한게 그럼 원본 테이블에 인덱스가 5개 달려있었다면 어떻게 해야 되나요? 카피 테이블에 연동 해줘야 되는거죠?
이름만 원본 테이블이랑 같게 한다고 해서 되는건 아닐테니까요
